app/assets/javascripts/modules/banner.coffee

```diff
@@ -27,7 +27,7 @@
       $banner.data('banner.persist', options.persist)
       $('body').after($banner)
       # Do this in a timeout so the element has time to be placed in the DOM and animations can happen
-      setTimeout((-> $banner.removeClass('banner-hidden')), 0)
+      setTimeout((-> $banner?.removeClass('banner-hidden')), 0)
```

Comment:
I have seen an error in the console twice. So I just added it there but I didn't investigate why the error occurs...